packages/app/src/app/pages/Sandbox/Editor/Workspace/items/GitHub/Git/Git.tsx

```diff
@@ -0,0 +1,156 @@
+import Margin from '@codesandbox/common/lib/components/spacing/Margin';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import GithubBadge from '@codesandbox/common/lib/components/GithubBadge';
+import Input, { TextArea } from '@codesandbox/common/lib/components/Input';
+import Notice from '@codesandbox/common/lib/components/Notice';
+import { githubRepoUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { observer } from 'mobx-react-lite';
+import React, { useEffect } from 'react';
+
+import { useSignals, useStore } from 'app/store';
+
+import { WorkspaceSubtitle, WorkspaceInputContainer } from '../../../elements';
+
+import { Container, Buttons, ErrorMessage, NoChanges } from './elements';
+import { TotalChanges } from './TotalChanges';
+
+function hasWriteAccess(rights: 'none' | 'read' | 'write' | 'admin') {
+  return rights === 'write' || rights === 'admin';
+}
+
+export const Git = observer(() => {
+  const {
+    git: {
+      createCommitClicked,
+      createPrClicked,
+      descriptionChanged,
+      gitMounted,
+      subjectChanged,
+    },
+  } = useSignals();
+  const {
+    editor: {
+      currentSandbox: { originalGit },
+      isAllModulesSynced,
+    },
+    git: { description, isFetching, originalGitChanges: gitChanges, subject },
+  } = useStore();
+
+  useEffect(() => {
+    gitMounted();
+  }, []); // eslint-disable-line
```

Comment:
@Saeris It's a best practice to add the `gitMounted` function to the `useEffect` dependencies even though you only want the effect to run on mount

```suggestion
  }, [gitMounted]);
```